cypress/e2e/playground.cy.ts

```diff
@@ -0,0 +1,87 @@
+const getIframeBody = () => {
+    return cy
+      .get('iframe[id="FCL_IFRAME"]')
+      .its('0.contentDocument.body').should('not.be.empty')
+      .then(cy.wrap)
+  }
+  
+describe('Flow-Playground frontend tests', () => {
+
+    beforeEach(() => {
+        cy.viewport('macbook-16')
+        cy.visit('http://localhost:3000/')
+        cy.waitForReact(1000, '#root')
+        
+    })
+
+    it('deploys a contract', () => {
+        cy.react('AccountList').children().last().children().should('have.length', 5).first().click()
+        cy.react('ActionButton', {props: {type: 1}}).contains('Deploy').click()
+        cy.react('ActionButton', {props: {type: 1}}).contains('Redeploy')
+        cy.react('RenderResponse', {props: {resultType: 'CONTRACT'}}).should('exist')
+    })
+
+    it('sends a transaction', () =>{
+        cy.react('MenuList', {props: {title: 'Transaction Templates'}}).children().first().contains('Transaction')
+        cy.get('div[title=Transaction]').click()
+        cy.react('EditorTitle', {props: {type: 2}}).contains('Transaction Template')
+        // Ensure action button is disabled when contract not deployed yet
+        cy.react('ActionButton', {props: {type: 2, active: false}}).contains('Send')
+
+        // deploy contract
+        cy.react('AccountList').children().last().children().first().click()
+        cy.react('ActionButton', {props: {type: 1}}).click()
+        cy.wait(5000)
```

Comment:
Waiting can be avoided by making an assertion on the page content. Cypress will automatically wait for that assertion and fail if it doesn't pass